### ISSUE TITLE
Maximo Report Request [New] SBO Winter Weather: Sanding Routes WO Set Up

### ISSUE THREAD

**Issue by YakshDipa** [cityofaustin/atd-data-tech#14128]
Winter Weather Planning : Figure Out WOs for preplanned routes

**Comment by adry-martinez** [comment]
Pending to collect feedback from OPM and District Maintenance team this week (11/27 - 12/1)

**Comment by adry-martinez** [comment]
**January 8**
- [ ] Confirm with Dave A. that the sanding routes previously given for exercise is accurate
- [ ] Reach out to David G. to get street segments

**Comment by adry-martinez** [comment]
**1-19**
Met with David G. to discuss tabular data needed to upload sanding routes to MAXIMO.
Joined data between Maximo and GIS is MXASSETNUM. Pending data from David G.

**Comment by dianamartin** [comment]
# 1/24/24
Project Update
- [ ] Waiting on Geo for street segments data
- [ ] Waiting on CTM for report creation

**Comment by adry-martinez** [comment]
### 1/31/24
Discussed with CTM Nick O. on data requirements for creating BIRT report, currently in progress. 
CTM will be prioritizing this task as it was mentioned that leadership team are looking to review this report.

**Comment by adry-martinez** [comment]
**Feb 24, 2024**
Issues with queries, CTM Nick is still working on this.

**Comment by ReneVarg25** [comment]
### 3/11/2024
Nick is reporting a data error when trying to run within Maximo Prod
He is unable to pull the parent-child WOs.

**Comment by rebekkabry** [comment]
# 5/20/24 Notes with CTM 

- Getting a different outcome from the test environment, believes it is a document error. 
- Need to look into the data, could be the workgroup. For instance, if you take out the data for the workgroup it messes it up or work order relationships...running the report from MAXIMO failed = getting an error. 
- Leaving in Nick's queue to address, more troubleshooting needs to be performed. We think it has to do with the workgroups.

**Comment by adry-martinez** [comment]
<img width="698" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNG85QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4ebf5a5ca43cf616df5f086bc26f997521c106ed/image.png" alt="image.png" />

<img width="1276" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNHM5QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4dea121eee295418970c976d7e3081fd59c5cfcf/image.png" alt="image.png" />

<img width="1424" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNHc5QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3c9f8b89398102b6098c616c7f4368ebb966a068/image.png" alt="image.png" />

**Comment by adry-martinez** [comment]
@renevarg25

Provide Ganesh 5 work orders updated with either Magnesium or Calcium AND Street Temperature.

**Comment by rebekkabry** [comment]
@ReneVarg25 is this still something we need to create/or provide?

**Comment by ReneVarg25** [comment]
@rebekkabry I am unsure.

**Comment by rebekkabry** [comment]
# 11/14/24 Ops SBO Weather Ideas

- weather prep 
 - start of winter weather today

-> would like to test the workload preplanned for 2. routes 
(using a map from his own machine) 
A.  bridges wil need to be treated about 150 
 - running them on 6 routes, because they have s sprayers. (they will pretreat the bridges) as the event continuest hey will go out and check the status, do temp measures, retreat if necceassry with materials. (the dots) 
.Routes A-F 
.Need 135 work orders
.Description: Bridges [route], name of location of bridge. 

During an event (how information will be used) 
 - start with pretreatmment on the bridge. (if wet, we go into treatment mode) 
 - the crews will ocmmunicate via radio on the service center they deployed out. they will communicate the route and status of bridge. communicate the activities and findings (temp, materials). 
 - the manager of the bridge group at center will collect data and add into maximo. 
 - at the dep oper center would like report, they have access to maximo. 

B. PROACTIVE ROADWAY TREATMENT 
8 different routes, would remove the snow in one lane mile each direction. 300 miles of roadway clearnace.
- we would PROACTIVELY work these (not from 311) 

- these routes are concreated with GIS segments. Dave wants to use the top segments and then N or S. > build off of the road network. THE OBJECTID - should be the street segment number. 
 
20-30 teams 
each gruop has a lead 
each group 3 convo. 


NEED TO ASK DAVE: Work Orders Considerations: 
-Routes need specification? 
-Parents for Routes? PREP > BRIDGE/PROACTIVE ROADWAYS; RESPONSE > 
PREP > BRIDGE > ROUTEA/B/C
PREP > PROACTIVE ROADWAY > ROUTEA/B/C
RESPONSE > 
-Job Plan set up for material types? 
  >> Materials: 
  >> Equipment: 
  >> Staff: 
-Reports Needed (bridges/status) (confirm that shows up on the current report) 
  --> last time bridge was checked, overall status; Bridge assets and EOC matches - Bridge IDs are being used from GIS dataset. 
- get the materials names used to add to current reports. 
- request that dave create test work orders for each type of A and B work order examples. 
- Maximo segments to your map, assets and mutli assets? THE OBJECTID - should be the street segment number. 
- How do we currently differentate btw directions? (N/S) [one-way, orientation]
- CONCERNED: paper transmission via radio? 
- CONCERNED: Bridges need to be assets in Maximo, not currently. but could use lat and long, GPS ideas, Maximo ID. 


GOAL TO TEST IDEAS: 
1.  can we get the data directly comes in from the field? (radio from field > call into DOC > manually enter data into maximo (prob will worksheets), 

 - work log, actuals, 

2. working links, btw parties.

**Comment by ReneVarg25** [comment]
### 11/21/24
https://github.com/cityofaustin/atd-data-tech/issues/19937

**Comment by rebekkabry** [comment]
## 11/26/24 

- Sent Dave and Richard, updated report layout 
- Sent Dave and Richard (11/18 week) Job Plan

**Comment by rebekkabry** [comment]
Pending updates from SBO and/or Dave regarding work order import and creation needs. Blocked until further feedback.

**Comment by ReneVarg25** [comment]
### 1/21/25
@rebekkabry Does this still need??

**Comment by dianamartin** [comment]
# 3/7/25
I think this can be closed @rebekkabry what do you think?

**Comment by rebekkabry** [comment]
@dianamartin Yes it can be. There are new issues that represent the current status and plans for this project/initative.
